reference/redirects.v3.yml

```diff
@@ -155,6 +156,215 @@ paths:
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/301RedirectImportExportJobRead'
+                  meta:
+                    $ref: '#/components/schemas/MetaPaginationObject'
+  '/storefront/redirects/imex/export':
+    parameters:
+      - $ref: '#/components/parameters/Accept'
+    post:
+      tags:
+        - Redirect Import-Export
+      summary: Create Redirects Export Job
+      description: Creates a new 301 Redirects export job.
+      operationId: createRedirectExportJob
+      parameters:
+        - $ref: '#/components/parameters/ContentType'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                site_id:
+                  type: integer
+                  nullable: true
+                  default: null
+                  description: The site ID for which you wish to export redirects. If no site ID is provided, the request exports all redirects for all sites.
+                redirect_ids:
+                  type: array
+                  items:
+                    type: integer
+                  default: []
+                  description: A list of the redirect IDs you wish to export. If no redirect IDs are provided, the request exports all redirects for the given site selection.
+              description: Data necessary to create a new 301 Redirects export job.
+        required: true
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  id:
+                    type: string
+                    example: ab1c2de3-f4gh-5678-i90j-klm12n345o67
+        '429':
+          description: Your store already has an active Redirects import or export job running.
+        '409':
+          description: No Redirects in your store meet the criteria in your request.
+  '/storefront/redirects/imex/import':
+    parameters:
+      - $ref: '#/components/parameters/Accept'
+    post:
+      tags:
+        - Redirect Import-Export
+      summary: Create Redirects Import Job
+      description: Creates a new 301 Redirects import job.
+      operationId: createRedirectImportJob
+      parameters:
+        - $ref: '#/components/parameters/ContentTypeFormData'
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                import_file:
+                  description: A CSV file containing a list of Redirects to be imported.
```

Comment:
can you say more about the structure of the CSV file?

Reply:
@slsriehl Sure, the CSV file must have the following headers:
```
Domain,Old Path,New URL/Path,Target Type,Target ID
```

And here's an example:
```
Domain,Old Path,New URL/Path,Target Type,Target ID
your-store.com,/old-path,/new-manual-path,,
your-store.com,/old-product,,Product,12
your-store.com,/old-brand,,Brand,34
your-store.com,/old-category,,Category,56
your-store.com,/old-page,,Page,78
your-store.com,/old-post,,Post,90
```